examples/browser-to-server/package.json

```diff
@@ -5,14 +5,20 @@
   "type": "module",
   "scripts": {
     "start": "vite",
-    "go-libp2p-server": "cd ../go-libp2p-server && go build && ./go-libp2p-server"
+    "build": "vite build",
+    "go-libp2p-server": "cd ../go-libp2p-server && go build && ./go-libp2p-server",
```

Comment:
Why actually going through the trouble of building it and running it, instead of just using `go run ./main.go` or w/e the binary entry is?

Reply:
Thanks Glen, I added this change.